_inc/client/plans/plan-grid.jsx

```diff
@@ -331,8 +332,19 @@ class PlanGrid extends React.Component {
 		const plan = this.getPlans()[ planType ];
 		const item = plan.features[ rowIndex ];
 		const key = planType + '-row-' + rowIndex;
+		const backupFeatureIds = [
+			'backups',
+			'malware-scan',
+			'real-time-backups',
+		];
+		const hideBackupFeature = (
+			! this.props.showBackups &&
+			item &&
+			-1 !== backupFeatureIds.indexOf( item.id )
```

Comment:
[Lodash's `includes` function](https://lodash.com/docs#includes) is already imported in this file so you can use it here:
```js
includes( backupFeatureIds, item.id )
```